website/templates/public/pages/help/registrations.mako

```diff
@@ -1,15 +1,17 @@
 <div class="col-md-12 col-sm-12">
     <span id="registrations" class="anchor"></span>
     <h4>Registrations</h4>
-    <p>Registrations are permanent, read only copies of a project. Registration saves the state of
-        a project at a particular point in time&mdash;such as right before data collection, or right when a manuscript
-        is submitted.</p>
-    <p>To register a project, click on the button in the grey navigation bar. Click on "New Registration", select a
-        meta-data template, fill it out, and then confirm the registration. The registration will at a separate,
+    <p>Registration creates a frozen version of the project that can never be edited or deleted but can be
+      retracted. You can register your project by selecting a registration form, entering information about
+      your project, and then confirming. You will be able to continue editing the original project, however,
+      and the frozen version with timestamps will always be linked to the original. Retracting a registration
+      will leave behind metadata about when the registration was created and retracted but removes the contents
+      of the registration.</p>
+
+    <p>A registration can be made public immediately or entered into an embargo period of up to 4 years. At the
+      end of the embargo period, the registration will automatically become public.</p>
+
+    <p>To register a project, click on the <em>Registrations</em> button in the grey navigation bar. Click on "New Registration", select a
+        meta-data template, fill it out, and then confirm the registration. The registration will be at a separate,
```

Comment:
Nods. That last bit was a remnant from the old instructions. Makes sense it
was outdated.

On Mon, Jun 15, 2015 at 4:15 PM, Steven Loria <notifications@github.com>
wrote:

> In website/templates/public/pages/help/registrations.mako
> <https://github.com/CenterForOpenScience/osf.io/pull/3142#discussion_r32461504>
> :
>
> > -        a project at a particular point in time&mdash;such as right before data collection, or right when a manuscript
> > -        is submitted.</p>
> > -    <p>To register a project, click on the button in the grey navigation bar. Click on "New Registration", select a
> > -        meta-data template, fill it out, and then confirm the registration. The registration will at a separate,
> > +    <p>Registration creates a frozen version of the project that can never be edited or deleted but can be
> > +      retracted. You can register your project by selecting a registration form, entering information about
> > +      your project, and then confirming. You will be able to continue editing the original project, however,
> > +      and the frozen version with timestamps will always be linked to the original. Retracting a registration
> > +      will leave behind metadata about when the registration was created and retracted but removes the contents
> > +      of the registration.</p>
> > +
> > +    <p>A registration can be made public immediately or entered into an embargo period of up to 4 years. At the
> > +      end of the embargo period, the registration will automatically become public.</p>
> > +
> > +    <p>To register a project, click on the <em>Registrations</em> button in the grey navigation bar. Click on "New Registration", select a
> > +        meta-data template, fill it out, and then confirm the registration. The registration will be at a separate,
>
> Will fix when I merge.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CenterForOpenScience/osf.io/pull/3142/files#r32461504>
> .
>



-- 
*Harry Rybacki* - Developer, Center for Open Science
Research: http://scinet.osf.io
Blog: http://rybacki.io
[image: http://github.com/harryrybacki]
<http://github.com/HarryRybacki> [image:
https://twitter.com/HarryRybacki] <https://twitter.com/HarryRybacki> [image:
http://www.linkedin.com/in/HarryRybacki/]
<http://www.linkedin.com/in/HarryRybacki/> [image:
https://plus.google.com/+HarryRybacki]
<https://plus.google.com/+HarryRybacki>